### PR TITLE
[DRE-973] Have ArgoCD deployments use Git instead of Chartmuseum

### PR DESCRIPTION
**Pull request by tochukwuvictor**

### CODE REVIEW

**Review by rkenney525 — APPROVED** [review]

File: .github/workflows/k8s-deploy-prod.yaml [hunk 1]
```diff
@@ -120,25 +120,24 @@ jobs:
           chartname="$( ./yq e '.name' Chart.yaml )"
           helm fetch chartmuseum/"$chartname" --version="${LATEST_HELM_CHART_VERSION}" --destination tmp --untar
           for i in staging prod; do
-            live_deploy_tag="$( ./yq e '.sb-service.image.tag' "tmp/$chartname/$i.yaml" )" \
-              ./yq e '.sb-service.image.tag = env(live_deploy_tag)' -i "$i.yaml"
+            live_deploy_tag="$( ./yq e ".$HELM_VALUES_FILE_IMAGE_TAG_PATH" "tmp/$chartname/$i.yaml" )" \
+              ./yq e '.'$HELM_VALUES_FILE_IMAGE_TAG_PATH' = env(live_deploy_tag)' -i "$i.yaml"
```
Comment (jgillis01) [inline .github/workflows/k8s-deploy-prod.yaml:124]: How is the `.` inside the quotes different than outside above it?

**Review by jgillis01 — APPROVED** [review]

File: .github/workflows/k8s-deploy-prod.yaml (hunk 1 above)
Comment (tochukwuvictor) [reply .github/workflows/k8s-deploy-prod.yaml:124]: Shell variable interpolation is weird. Would you prefer `''".$HELM_VALUES_FILE_IMAGE_TAG_PATH"'` over `'.'$HELM_VALUES_FILE_IMAGE_TAG_PATH'`?

Comment (jgillis01) [reply .github/workflows/k8s-deploy-prod.yaml:124]: lol....no, and thank you

Comment (tochukwuvictor) [reply .github/workflows/k8s-deploy-prod.yaml:124]: :heh: np

**Pull request merged**